### DISCUSSION TITLE
UTM Stack wineventlogs not coming through

### DISCUSSION THREAD

**Discussion by AndrewG98** [utmstack/UTMStack#850] [Q&A] [answered]
Good Day, please note I have logged into my UTM stack console today and I went to my data sources on utm stack and I can see an alert on all my servers I have listed on my utm stack where the alert says wineventlog is taking longer than usual to send logs , also if I go to my log explorer, I see no logs there as well, please can someone help me.
![utm](https://github.com/user-attachments/assets/4fc63b27-f16e-45f6-abbc-da820588735f)

**Comment by c3s4rfred** [comment, marked as answer]
Hi @AndrewG98 check your installed windows agent, maybe you have changed the machine's IP or you have a network issue. Check the connection between agent's machine and UTMStack's machine. You can also check the state of the services below:
![image](https://github.com/user-attachments/assets/2769d403-6488-4ec0-8dc8-3a64304ab182)
![image](https://github.com/user-attachments/assets/4e9364f0-ae13-4aba-915d-2d6bc43ef643)
and see the agent log at: C:\Program Files\UTMStack\UTMStack Agent\logs

Best Regards
